src/install/index.js

```diff
@@ -268,11 +272,25 @@ const predeploySteps = (deployDoc) => {
     .then(() => stage('horti.stage.extractingDdocs', 'Extracting ddocs'))
     .then(() => extractDdocs(ddoc))
     .then(() => stage('horti.stage.warmingViews', 'Warming views'))
-    .then(() => warmViews(deployDoc))
+    .then(() => warmViews(deployDoc, true))
     .then(() => stage('horti.stage.readyToDeploy', 'View warming complete, ready to deploy'))
     .then(() => ddoc);
 };
 
+const unstagedPredeploySteps = (deployDoc) => {
```

Comment:
I don't think you need this at all, because warming views doesn't make sense in satellite mode, as production locations are being overridden. In fact, this is a generally problematic issue with this mode.

In normal mode:
 - You are running version X
 - A deploy command comes in to deploy to version Y
 - horti downloads the build to a staged location, as well as extracting any additional design documents into staged locations
 - The app is still running X, all exposed ddocs etc are still version X
 - horti "warms" Y's views by running an query against them
 - Once that is done, horti is confident it can:
   - kill api and sentinel
   - move Y ddocs over X ddocs
   - move the symlink (where required) from X to Y for api and sentinel
   - start api and sentinel again, and have the views ready to go

This works because view index data is based on a hash key of the view code itself, which means the design doc name doesn't matter, which means the indexes generated from `:staged:` design documents can work with their production location copies.

In satellite mode however, this no longer makes sense, because we're listening to production data being replicated into production locations.

It's also more confusing when combined with what to watch, as described above.

Reply:
Let's discuss on our call. But imagine (10GB?) a large primary instance in stable state (no deployment). Now start a satellite which replicates this data. My intention here is that these replicated views on the new satellite needs to be warmed. So this change is targeting a scenario unrelated to deployments.